src/modules/Dashboard/components/FlowSummaryCard/FlowSummaryCard.tsx

```diff
@@ -0,0 +1,53 @@
+import { Box } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+import { ActionsMenu, Chip, ChipShape, Svg } from 'shared/components';
+import { StyledEllipsisText, StyledFlexColumn, variables } from 'shared/styles';
+import { getDictionaryText } from 'shared/utils';
+
+import { FlowSummaryCardProps } from './FlowSummaryCard.types';
+import { StyledRoot } from './FlowSummaryCard.styles';
+import { FlowSummaryThumbnail } from './FlowSummaryThumbnail';
+
+export const FlowSummaryCard = <T,>({
+  activities = [],
+  description,
+  menuItems,
+  name,
+  ...otherProps
+}: FlowSummaryCardProps<T>) => {
+  const { t } = useTranslation('app');
+
+  return (
+    <StyledRoot {...otherProps}>
+      <FlowSummaryThumbnail activities={activities} />
+
+      <StyledFlexColumn sx={{ gap: 0.8, flexGrow: 1, overflow: 'hidden' }}>
+        <StyledEllipsisText sx={{ fontSize: variables.font.size.xxl }}>{name}</StyledEllipsisText>
+
+        <StyledEllipsisText
+          sx={{ color: variables.palette.on_surface_variant, fontSize: variables.font.size.lg }}
+        >
+          {getDictionaryText(description)}
+        </StyledEllipsisText>
+      </StyledFlexColumn>
+
+      <Box sx={{ display: 'flex', gap: 0.8 }}>
+        <Chip
+          color="primary"
+          icon={<Svg aria-hidden id="logic" height={18} width={18} />}
+          shape={ChipShape.Rectangular}
+          title={t('flow')}
+        />
+
+        <Chip
+          icon={<Svg aria-hidden id="checklist-filled" height={18} width={18} />}
+          shape={ChipShape.Rectangular}
+          title={t('activityCount', { count: activities?.length ?? 0 })}
+        />
```

Comment:
Nit: just noticing that Chips are a slightly different height in the `square` variant vs. `rounded`:

<img width="189" alt="Screenshot 2024-05-01 at 6 48 43 PM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/774fe270-19bf-4f9d-9aeb-ea47768d2773">
<img width="206" alt="Screenshot 2024-05-01 at 6 48 48 PM" src="https://github.com/ChildMindInstitute/mindlogger-admin/assets/1282772/f9bfd151-5b63-4f8c-bd45-0452ab5f3ff9">

I guess to make them align to Figma, you'd have to remove the static `height` and `lineHeight` as dictated in `theme.tsx`, and instead change `line-height` dynamically in `Chip.styles.ts` based on the `shape` prop (`variables.font.lineHeight.md` vs `variables.font.lineHeight.lg`).

Only thing is, if you make this change to all square chips, it's going to affect the changes I made in #1687 for the Participants autocomplete field on the Add Team Member screen, where we rely on an expected height of square chips to limit the height of the autocomplete field to 3 rows. Also maybe it will affect other screens unpredictably… hard to stay without some digging.

Maybe don't worry about it for now. I'm being too eagle-eyed! 🦅

Reply:
Yeah, I left this alone for now, mostly to minimize potential disruption. This seems worthwhile as a standalone task.